server/configmodule.py

```diff
@@ -54,10 +54,12 @@ class StagingConfig(Config):
 class AutopushConfig(Config):
     GA_ACCOUNT = 'G-Y6ZXZ9JK3H'
     ADMIN = True
+    GCS_BUCKET = 'datcom-website-autopush-resources'
     pass
 
 
 class DevConfig(Config):
+    GCS_BUCKET = 'datcom-website-autopush-resources'
```

Comment:
ideally this should be in dev? I guess you need some cross gcp IAM permission for robot account if doing this.

Reply:
do you mean we would need cross gcp IAM permission for robot account if putting the files in dev or if putting the files in autopush? Because right now, dev seems to work with this code?